packages/manager/src/features/Kubernetes/KubernetesClusterDetail/KubeConfigDisplay.tsx

```diff
@@ -96,7 +99,15 @@ export const KubeConfigDisplay = (props: Props) => {
   const { enqueueSnackbar } = useSnackbar();
   const { classes, cx } = useStyles();
 
-  const { refetch } = useKubenetesKubeConfigQuery(clusterId);
+  const { refetch, isFetching } = useKubenetesKubeConfigQuery(clusterId, false);
+
+  const onGetToken = async () => {
+    const { data } = await refetch();
+    const token = data && data.match(/token:\s*(\S+)/);
+    if (token) {
+      copy(token[1]);
+    }
+  };
```

Comment:
We should add some general error handling. For example:

```suggestion
  const { isFetching, refetch: getKubeConfig } = useKubenetesKubeConfigQuery(
    clusterId,
    false
  );

  const onGetToken = async () => {
    try {
      const { data } = await getKubeConfig();
      const token = data && data.match(/token:\s*(\S+)/);
      if (token && token[1]) {
        copy(token[1]);
      } else {
        enqueueSnackbar({
          message: 'Unable to find token within the Kubeconfig',
          variant: 'error',
        });
      }
    } catch (error) {
      enqueueSnackbar({
        message: (error as APIError[])[0].reason,
        variant: 'error',
      });
    }
  };
```